comp/dogstatsd/server/intern.go

```diff
@@ -6,93 +6,99 @@
 package server
 
 import (
+	"runtime"
+	"sync"
+	"unsafe"
+
 	"github.com/DataDog/datadog-agent/pkg/config/utils"
 	"github.com/DataDog/datadog-agent/pkg/telemetry"
-	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
 var (
 	// There are multiple instances of the interner, one per worker. Counters are normally fine,
 	// gauges require special care to make sense. We don't need to clean up when an instance is
 	// dropped, because it only happens on agent shutdown.
-	tlmSIResets = telemetry.NewSimpleCounter("dogstatsd", "string_interner_resets",
-		"Amount of resets of the string interner used in dogstatsd")
-	tlmSIRSize = telemetry.NewSimpleGauge("dogstatsd", "string_interner_entries",
-		"Number of entries in the string interner")
-	tlmSIRBytes = telemetry.NewSimpleGauge("dogstatsd", "string_interner_bytes",
-		"Number of bytes stored in the string interner")
```

Comment:
This is really really valuable imformation, can we try to keep it?

Reply:
yup, don't worry I'll make sure all existing telemetry is covered in a final version of this PR.